### ISSUE TITLE
Missing converters

### ISSUE THREAD

**Issue by perceptron8** [marschall/threeten-jpa#4]
How about including missing converters from [montanajava/jpaattributeconverters](https://bitbucket.org/montanajava/jpaattributeconverters)?

* DayOfWeekConverter
* DurationConverter
* InstantConverter
* MonthDayConverter
* MonthConverter
* OffsetDateTimeConverter
* OffsetTimeConverter
* PeriodConverter
* YearMonthConverter
* YearConverter
* ZoneIdConverter
* ZonedDateTimeConverter

**Comment by perceptron8** [comment]
I couldn't wait, I created my own project. Tried forking, but ended with total rewrite.

There are more converters (some of them are optional). Conversions are much simpler (I used ```LocalDate|Time.valueOf()``` methods instead of dirty ```Calendar``` tricks). I rewrote tests too (they are more exhaustive, all coming to my mind edge cases are tested).

You are very welcome to borrow every single line of code if you like.

https://github.com/perceptron8/datetime-jpa

PS. First release is now available in maven central!
